examples/example-dsl/src/main/scala/com/lightbend/kafka/scala/iq/example/WeblogProcessing.scala

```diff
@@ -123,8 +124,9 @@ object WeblogProcessing extends WeblogWorkflow with AppSerializers with FailFast
 
     //
     // assumption : the topic contains serialized records of LogRecord (serialized through logRecordSerde)
-    val logRecords = 
-      builder.stream(List(config.toTopic), Consumed.`with`(byteArraySerde, logRecordSerde))
+    // will use default serializers from config
+
+    val logRecords = builder.stream[Array[Byte], LogRecord](config.toTopic)
```

Comment:
This will use default serde from config since the implicit for `LogRecordSerde` is not in scope.